starter-code/index.html

```diff
@@ -3,40 +3,66 @@
 <head>
   <meta charset="UTF-8">
   <title>Spotify</title>
+  <link rel="stylesheet" href="../starter-code/css/main.css" />
+
 </head>
 <body>
-Premium Discover Help Download
-
-Music for everyone.
-Spotify is now free on mobile, tablet and computer.
-Listen to the right music, wherever you are.
-
-What’s on Spotify?
-
-Millions of Songs
-There are millions of
-songs on Spotify
-
-HD Music
-Listen to music as if you
-were listening live
-
-Stream Everywhere
-Stream music on your
-smartphone, tablet or
-computer
+  <nav class="nav-style">
+    <img src="/home/valentin/code/lab/lab-css-spotify-clone/starter-code/images/spotify-logo.png" class="animation-logo" alt="logo">
```

Comment:
Same comment as before: use a relative path instead. When you use absolute paths it only works on your computer; as soon as you put your page online everything will break.